pkg/minishift/network/validations/validations.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validations
+
+import (
+	"fmt"
+	"strings"
+
+	minishiftConstants "github.com/minishift/minishift/pkg/minishift/constants"
+	"github.com/minishift/minishift/pkg/minishift/shell/powershell"
+)
+
+var VMSwitch string
```

Comment:
@anjannath this doesn't make much sense, if you check your `IsUsingDefaultSwitch` implementation then it would be good to pass the switchName (whatever it will be) then evaluate it.

Reply:
@praveenkumar if you mean instead of using this variable just pass the value we get from `viper.GetString(configCmd.HypervVirtualSwitch.Name)` ?